README.md

```diff
@@ -4,7 +4,7 @@
 
 This is DocBot, a sample Teams bot implementation that simulates scheduling a Doctor's appointment.
 
-This project represents a scenario where a backend API handles all of the dialog flow logic. All user responses are forwarded to the API and all of the API's responses are transformed/adapted to be displayed to the user as [Adaptive Cards](https://adaptivecards.io/). To achieve this, the bot leverages `adaptive-card-transformer`. See the [Adaptive Card Transformer repo](https://github.com/retaildevcrews/AdaptiveCardTransformer) for more information.
+This project represents a scenario where a backend API handles all of the dialog flow logic. All user responses are forwarded to the API and all of the API's responses are transformed/adapted to be displayed to the user as [Adaptive Cards](https://adaptivecards.io/). To achieve this, the bot leverages the [Adaptive Card Tranformer](https://github.com/retaildevcrews/AdaptiveCardTransformer).
 
```

Comment:
Do we perhaps want to link to the `How to Integrate` file in the `AdaptiveCardTransformer` somewhere in the README just to make it super discoverable?